ydb/docs/en/core/integrations/grafana.md

```diff
@@ -0,0 +1,136 @@
+# {{ ydb-short-name }} data source for Grafana
+
+The [{{ ydb-short-name }} data source plugin](https://grafana.com/grafana/plugins/ydbtech-ydb-datasource/) allows you to use [Grafana](https://grafana.com) to query and visualize data from {{ ydb-short-name }}.
+
+## Installation
+
+Prerequisites: the plugin requires `v9.2` and higher of Grafana.
+
+Follow the Grafana's [plugin installation docs](https://grafana.com/docs/grafana/latest/plugins/installation/) to install a plugin named `ydb-grafana-datasource-plugin`.
+
+## Configuration
+
+### {{ ydb-short-name }} user for the data source
+
+Set up an {{ ydb-short-name }} user account with **read-only** permissions [(more about permissions)](../cluster/access.md) and access to databases and tables you want to query. 
+
+{% note warning %}
+
+Please note that Grafana does not validate that queries are safe. Queries can contain any SQL statements including data modification instructions.
+
+{% endnote %}
+
+### Data transfer protocol support
+
+The plugin supports [gRPC and gRPCS](https://grpc.io/) transport protocols. A TLS/SSL certificate needs to be provided when using `grpcs://`.
+
+### Configuration via UI
+
+Once the plugin is installed on your Grafana instance, follow [these instructions](https://grafana.com/docs/grafana/latest/datasources/add-a-data-source/) to add a new {{ ydb-short-name }} data source, and enter configuration options.
+
+### Configuration with provisioning system
+
+Alternatively, it is possible to configure data sources using configuration files with Grafana’s provisioning system. To read about how it works, including all the settings that you can set for this data source, refer to [Provisioning Grafana data sources](https://grafana.com/docs/grafana/latest/administration/provisioning/#data-sources) documentation.
+
+### Authentication
+
+The Grafana plugin supports various authentication [authentication methods](../reference/ydb-sdk/auth.md).
+
+Below is an example config for authenticating a {{ ydb-short-name }} data source using username and password:
+
+```yaml
+apiVersion: 1
+datasources:
+  - name: YDB
+    type: ydbtech-ydb-datasource
+    jsonData:
+      authKind: "UserPassword"
+      endpoint: 'grpcs://<hostname>:2135'
+      dbLocation: '/eu-central1/b1g/db-location'
+      user: '<username>'
+    secureJsonData:
+      password: '<userpassword>'
+      certificate: 'content of *.pem file'
```

Comment:
where to get this pem file?

Reply:
We may give a link to https://ydb.tech/docs/en/concepts/connect#tls-cert